package.json

```diff
@@ -14,7 +14,6 @@
     "eslint-plugin-babel": "^4.0.0",
     "eslint-plugin-flow-vars": "^0.5.0",
     "eslint-plugin-flowtype": "^2.28.2",
-    "eslint-plugin-jest": "file:packages/eslint-plugin-jest",
```

Comment:
This causes linting to fail on Windows.  Does `eslint` search for plugins differently on Unix systems?